resume/src/app/navigation-bar/navigation-bar.component.html

```diff
@@ -0,0 +1,26 @@
+<nav class="flex sticky bg-gray-light items-center justify-between flex-wrap p-2 top-0 z-10">
+    <div class="flex items-center flex-shrink-0 text-black hover:text-white mr-6 ps-10">
+        <a routerLink="/" class="font-semibold text-xl tracking-tight">
+            Home
+        </a>
+    </div>
+
+    <div class="text-sm flex-grow flex-grow tems-center w-auto">
+        <a routerLink="/cv" class="inline-block mt-0 hover:text-white mr-4">
+            CV
+        </a>
+        <a routerLink="/projects" class="inline-block mt-0 hover:text-white mr-4">
+            Projects
+        </a>
+    </div>
+    
+    <div class="text-sm w-auto text-gray inline-flex">
```

Comment:
I also wanted to add a tooltip on hover here to explain what the link does, but at the moment, only the "title" attribute shows up after a couple of seconds (and not very stylised)